src/kex_ntru/ntru_crypto_drbg.c

```diff
@@ -347,9 +348,9 @@ sha256_hmac_drbg_instantiate(
 	return result;
 }
 
-/* sha256_hmac_drbg_free
+/* sha256_hmac_drbg_free IGNORE free-check
  *
- * This routine frees a SHA-256 HMAC_DRBG internal state.
+ * This routine frees a SHA-256 HMAC_DRBG internal state. IGNORE free-check
```

Comment:
Are these here because the script is catching the use of the word free even though it's in a comment?  If so is there any change we can make to the regular expression to avoid this?  E.g. have the regular expression look for `free[:space:]*(` since a call to free would require a parentheses after the word free (possibly separated by some whitespace).